src/pages/Register/Register.js

```diff
@@ -39,9 +40,12 @@ export const Register = () => {
         <div className="Register">
             <h1>Súmate a la mejor plataforma de música</h1>
             <p>Accede a millones de canciones, de todos los estilos, en todos tus dispositivos. Solo te llevará 30 segundos.</p>
-            <input type="text" name="name" onChange={handleInput} ref={register({ required: true, maxLength: 8 })} placeholder="Name" />
-            <input type="email" name="email" onChange={handleInput} ref={register({ required: true })} placeholder="Email" />
-            <input type="password" name="password" onChange={handleInput} ref={register({ pattern: /^[A-Za-z0-9]+$/i })} placeholder="Password" />
+            <Input type={"text"} name={"name"} onChange={handleInput} placeholder={"Name"}/>
+            <Input type={"email"} name={"email"} onChange={handleInput} placeholder={"E-mail"}/>
+            <Input type={"password"} name={"password"} onChange={handleInput} placeholder={"Password"}/>
+            {/* <input type="text" name="name" onChange={handleInput} ref={register({ required: true, maxLength: 8 })} placeholder="Name" /> */}
```

Comment:
If the old input component is being replaced by yours feel free to erase old calls, no need to comment them